actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add `render json` into `HealthController`.
```

Comment:
We can improve this a bit, perhaps by describing the purpose of the changes instead of pointing out directly what changes were made in the code.